chroma_cli/api.py

```diff
@@ -78,18 +78,20 @@ def start_session(self):
             raise RuntimeError("No session (status: %s, text: %s)" % (r.status_code, r.content))
 
         self.session.headers["X-CSRFToken"] = r.cookies["csrftoken"]
-        self.session.cookies["csrftoken"] = r.cookies["csrftoken"]
-        self.session.cookies["sessionid"] = r.cookies["sessionid"]
+        self.session.cookies.set("csrftoken", r.cookies["csrftoken"])
 
     def login(self, **credentials):
         if not self.is_authenticated:
             if "sessionid" not in self.session.cookies:
                 self.start_session()
 
             r = self.post(self.session_uri, data=json.dumps(credentials))
+
             if not 200 <= r.status_code < 300:
                 raise AuthenticationFailure()
             else:
+                self.session.headers["X-CSRFToken"] = r.cookies["csrftoken"]
```

Comment:
Should this also use r.cookies.get(...)?

Reply:
Doesn't matter.